### ISSUE TITLE
HW8 Data labels

### ISSUE THREAD

**Issue by laurenjli** [Computational-Content-Analysis-2020/frequently-asked-questions#43]
Hi,

Similar to HW3, it looks like HW8 requires the corpus to have classification labels. What should we do if we don't have a lot of labels for our corpus? Should we run the exercises on one of the corpus examples already in the notebook?

Thanks!

**Comment by bhargavvader** [comment]
Yes you can, or you can download a dataset with labels online. [This tutorial](https://medium.com/huggingface/multi-label-text-classification-using-bert-the-mighty-transformer-69714fa3fb3d) is one such example. Let me know if you'd need anything else.
